src/include/rear/ui.rb

```diff
@@ -184,17 +201,90 @@ def DirectoriesDialog(directories)
       deep_copy(directories)
     end
 
-    def SaveConfig(modules_load, backup_prog_include)
-      modules_load = deep_copy(modules_load)
-      backup_prog_include = deep_copy(backup_prog_include)
-      Rear.modified = true
-      Rear.output = Convert.to_string(UI.QueryWidget(Id(:output), :Value))
-      Rear.netfs_url = Convert.to_string(UI.QueryWidget(Id(:netfs_url), :Value))
-      Rear.netfs_keep_old_backup = Convert.to_boolean(
-        UI.QueryWidget(Id(:netfs_keep_old_backup), :Value)
+    def ListEditDialog(title, list)
+      # store original list for the case that the users clicks cancel
+      list_sav = list
```

Comment:
in ruby this do nothing. it in fact just store reference. If you want original content, then you need to use deep_copy